CMakeLists.txt

```diff
@@ -450,7 +450,7 @@ if(ANDROID)
 	SET(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY)
 	SET(CMAKE_FIND_ROOT_PATH_MODE_PROGRAM NEVER)
 	SET(CMAKE_FIND_ROOT_PATH_MODE_PACKAGE ONLY)
-elseif(NOT APPLE AND NOT CMAKE_SYSTEM_NAME MATCHES OpenBSD)
+elseif(CMAKE_SYSTEM_NAME STREQUAL "Linux")
```

Comment:
Definitely, but I wanted a correct and simple quickfix for this.
Right now, it's kind of hard to make sense of all of what is in this file until we remove a lot of stuff in there.